wallet/notifications.go

```diff
@@ -401,6 +452,33 @@ type Block struct {
 	Transactions []TransactionSummary
 }
 
+// TicketSummary contains the properties to describe a ticket's current status
+type TicketSummary struct {
+	Ticket  *TransactionSummary
+	Spender *TransactionSummary
+	Status  TicketStatus
+}
+
+// TicketStatus describes the current status a ticket can be observed to be.
+type TicketStatus int8
```

Comment:
we may need an "unknown" value as well.  This will allow the ticket results to still be created even if not all info regarding its state can be determined.  See the comment about returning nil and the panic above.